client/src/shared/utils/helpers.ts

```diff
@@ -43,3 +43,7 @@ export const isAuthorized = (is_trainer: boolean, pathname: string): boolean =>
   const regex = new RegExp(`^/(${slug}.+)`);
   return regex.test(pathname);
 };
+
+export const isYoutubeLink = (link: string): boolean => {
+  return link.startsWith('https://www.youtube.com');
```

Comment:
Maybe you can try to use regular expressions to validate a YouTube link.

Try this regex:
`/^(?:https?:\/\/)?(?:m\.|www\.)?(?:youtu\.be\/|youtube\.com\/(?:embed\/|v\/|watch\?v=|watch\?.+&v=))((\w|-){11})(?:\S+)?$/`